Cargo.toml

```diff
@@ -44,3 +45,4 @@ esp-idf-sys = { version = "0.32", features = ["native", "binstart"] }
 mipidsi = "0.5.0"
 display-interface-spi = "0.4.1"
 embedded-graphics = "0.7.1"
+nb = "1.0.0"
```

Comment:
Where is this used?

Reply:
This was likely an editor screwup (auto-import/use).  I'll fix it, and try to figure out what happened.